js/make.js

```diff
@@ -102,6 +104,22 @@ b.task('test', () => {
   })
 })
 
+b.task('cover', () => {
```

Comment:
@oliver---- could you help here? this is producing coverage statistics but reporting against file `./tmp/test.cover.js`, not against `./src/**/*.js`